### PR TITLE
multiple days can be moved back or forth

### PR DESCRIPTION
**Pull request by MarcinSalamon**

### CODE REVIEW

File: src/app/routes/calendar/calendar-modal/calendar-modal.component.ts [hunk 1]
```diff
@@ -6,6 +6,9 @@ import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material';
   templateUrl: './calendar-modal.component.html',
   styleUrls: ['./calendar-modal.component.css']
 })
+/**
```
Comment (YamasakiRhys) [inline src/app/routes/calendar/calendar-modal/calendar-modal.component.ts]: Empty comments, please add or remove

File: src/app/routes/calendar/calendar.component.ts [hunk 2]
```diff
@@ -322,8 +323,20 @@ export class CalendarComponent implements OnInit, DoCheck {
     return dialogRef.afterClosed();
   }
 
-  moveEvents(decision) {
-    console.log(decision);
+  moveEvents(decision, event: CalendarEvent) {
```
Comment (YamasakiRhys) [inline src/app/routes/calendar/calendar.component.ts]: add NgDoc to your method

**Review by YamasakiRhys — CHANGES_REQUESTED** [review]

File: src/app/routes/calendar/calendar-modal/calendar-modal.component.ts (hunk 1 above)
Comment (MarcinSalamon) [reply src/app/routes/calendar/calendar-modal/calendar-modal.component.ts]: done

**Review by YamasakiRhys — APPROVED** [review]

**Comment by YamasakiRhys** [comment]
Nice changes.  Looks good.

**Pull request merged**